mllib/src/main/scala/org/apache/spark/mllib/fpm/PrefixSpan.scala

```diff
@@ -43,6 +45,8 @@ class PrefixSpan private (
     private var minSupport: Double,
     private var maxPatternLength: Int) extends Logging with Serializable {
 
+  private val maxSuffixesBeforeLocalProcessing: Long = 10000
```

Comment:
This is really `maxProjectedDBSizeBeforeLocalProcessing` since you are counting the number of total items in the projected db not just number of suffixes